app/src/main/java/com/duckduckgo/mobile/android/duckduckgo/ui/bookmarks/BookmarkModel.java

```diff
@@ -0,0 +1,99 @@
+package com.duckduckgo.mobile.android.duckduckgo.ui.bookmarks;
+
+import android.os.Parcel;
+import android.os.Parcelable;
+
+import com.duckduckgo.mobile.android.duckduckgo.domain.bookmark.Bookmark;
+
+import java.util.UUID;
+
+/**
+ * Created by fgei on 6/12/17.
+ */
+
+public class BookmarkModel implements Bookmark, Parcelable {
```

Comment:
I see other projects tend to add the -model suffix for the presentation layer and the -entity suffix for the data layer. I agree that the model suffix is not really readable and definitely annoying to use, we could call the interface just IBookmark and change BookmarkModel to simply Bookmark but I'm not a fan of altering names for interfaces. In this case also I don't think we want the same suffix, assume we are adding another data source for the bookmark like a database, then the BookmarkJsonEntity name is not really appropriate I think